app/config/variables.php

```diff
@@ -88,6 +88,15 @@
                 'question' => '',
                 'filter' => ''
             ],
+            [
+                'name' => '_APP_CONSOLE_WHITELIST_CODES',
```

Comment:
Because this is just an internal feature, I don't think we should officially expose it, as we won't be able to provide proper long term support for it.

Reply:
Alright will remove it 👍